package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "sass-lint 'src/**/*.scss' -v",
-    "prepublish": "npm run lint"
+    "prepublish": "npm run lint",
```

Comment:
We could auto build on if we want by updating this to : `npm run lint && npm run <gulp  build task here>` 
Not sure which gulp task we should map to an npm script but this would ensure we have a fresh compile in `/dist` before its published to npm for usage.

Reply:
@Ollo Will that matter if we aren't consuming it directly from the package? Seems like that step might bet better on post install, so you're not lugging the `dist` folder around. Either way, good call to automate that step (whenever the appropriate time is).

What are your thoughts on deployment to CDN?. If we're versioning, the build server could run the build, read package.json for the version number and then deploy to a versioned folder name on dugout, yeah?

Reply:
Since publishing has typically been the only spot where changes are consistently versioned I was thinking that might be the appropriate place to build the `dist` folder and copy it to the cdn in a versioned directory structure `dugout.teamsnap.com/teamsnap-ui/v2.4.0/`. ?

Reply:
For reference, when I think "publish" I think `npm publish`.... when I think deploy, I think `jarvis deploy...`.

I don't necessarily see a ton wrong with building `dist` on publish in that case (we do it with the SDK), but I'm curious if there will be env considerations re: deploy, that would cause the build to need to re-run, anyway.

Reply:
@Ollo @matthopson Here's what I ended up doing with scripts:

+ For deployment to CDN, we'll be telling Ops which task to run on `jarvis deploy`. I'm thinking `gulp build` is the right one, since it cleans out the `dist` directory and runs through all the build steps, without starting any servers or watching anything. 

+ Ops also said they could reference the version from the package.json for creating the directory structure on the CDN when we deploy. Awesome. Great idea @matthopson! 

+ Left `prepublish` untouched since that seems good as is. 

+ Added `gulp build` to `npm install` which will add all the `dist` goodness when you install the node module, should anyone want to reference or pull from these assets (don't think this will be a common use case, but figured it doesn't hurt)

+ `npm start` runs `gulp`, which in addition to `build`ing, starts a server and watches files. This is for local development from the repository.